pyproject.toml

```diff
@@ -43,6 +43,7 @@ types-python-dateutil = "2.9.0.20241003"
 types-jsonschema = "4.23.0.20240813"
 pytest = "8.3.3"
 coverage = "7.6.7"
+pytest-subtests = "^0.13.1"
```

Comment:
I would prefer to use a fixed version and let dependabot do the job. Otherwise we could get a conflict in the lock-file in some PRs, if the version of this package change.

Reply:
An oversight on my part. Will be fixed.